cli/stork-publisher-agent/processor_test.go

```diff
@@ -0,0 +1,115 @@
+package stork_publisher_agent
+
+import (
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+)
+
+const evmPrivateKey = "0x8b558d5fc31eb64bb51d44b4b28658180e96764d5d5ac68e6d124f86f576d9de"
+const evmPublicKey = "0x99e295e85cb07c16b7bb62a44df532a7f2620237"
+const starkPublicKey = "0x66253bdeb3c1a235cf4376611e3a14474e2c00fd2fb225f9a388faae7fb095a"
+const starkPrivateKey = "0x418d3fd8219a2cf32a00d458f61802d17f01c5bcde5a4f82008ee4a7c8e9a06"
+const storkAuth = "fake_auth"
+
+const assetId = "fakeAsset"
+
+func getNextSignedOutput[T Signature](ch chan SignedPriceUpdateBatch[T], timeout time.Duration) (SignedPriceUpdateBatch[T], bool) {
```

Comment:
helper function to get the next signed price, but time out after some amount of time so we don't wait infinitely in the failure case.